Branch-SDK/Branch-SDK/BNCSystemObserver.m

```diff
@@ -121,13 +121,16 @@ + (NSString *)getDeviceName {
 }
 
 + (NSNumber *)getUpdateState {
-    NSString *bundleRoot = [[NSBundle mainBundle] bundlePath];    NSFileManager *manager = [NSFileManager defaultManager];
```

Comment:
I like this, but I wanted to maintain whatever the original function signature was. Can our server accept BOOL values in the JSON? Havent tested it myself